packages/@divvi/mobile/src/public/navigate.ts

```diff
@@ -0,0 +1,94 @@
+// See useWallet for why we don't directly import internal modules, except for the types
+import type { FiatExchangeFlowType } from '../fiatExchanges/types'
+import type { Navigate } from '../navigator/NavigationService'
+import type { ScreensType } from '../navigator/Screens'
+import type { NetworkId as InternalNetworkId } from '../transactions/types'
+import type { NetworkId } from './types'
+
+declare global {
+  // eslint-disable-next-line @typescript-eslint/no-namespace
+  namespace DivviNavigation {
+    // eslint-disable-next-line @typescript-eslint/no-empty-interface
+    interface RootParamList extends StackParamList {}
+  }
+}
+
+export type StackParamList = {
+  Send: undefined
+  Receive: undefined
+  Swap:
+    | {
+        fromTokenId?: string
+        toTokenId?: string
+        toTokenNetworkId?: NetworkId
+      }
+    | undefined
+  Add: undefined
+  Withdraw: undefined
+  TabWallet: undefined
+  TabActivity: undefined
+  TabEarn: undefined
+  TabDiscover: undefined
```

Comment:
I was wondering if I should include the `Tab` prefix here.
Including it makes it clear that it switches tab, but forces us to keep tab navigation (unless we make a breaking change),